enos/ansible/roles/utils/tasks/ips.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Yes, in this case we should augment the Host structure returned by the init method.
This change is deep and could fit in a dedicated patch.